Scripts/header.sh

```
@@ -0,0 +1,104 @@
+  fi
+
+  # Check if the first line is the swift-format-ignore indicator
+  first_line=$(head -n 1 "$file")
+  if [[ "$first_line" == "// swift-format-ignore-file" ]]; then
+    echo "Skipping $file due to swift-format-ignore directive."
+    continue
+  fi
+
+  # Create the header with the current filename
+  filename=$(basename "$file")
+  header=$(printf "$header_template" "$filename" "$package" "$creator" "$year" "$company")
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)